src/_assets/css/_dash.scss

```diff
@@ -198,6 +198,9 @@ $dash-header-bg: #17212f;
   .banner__text {
     color: white;
   }
+  a {
+    color: #40c4ff;
```

Comment:
(nitpick) I suspect there's a variable with color defined somewhere

Reply:
I looked for this color and couldn’t find it, but yeah, it should be a variable.